operation/fhir-operation-everything/src/main/java/com/ibm/fhir/operation/everything/EverythingOperation.java

```diff
@@ -416,4 +470,283 @@ private static Uri uri(FHIROperationContext operationContext, String uriPath) {
+        } else if (compartmentMemberType.equals(Schedule.class.getSimpleName())) {
+            addSearchParameterIfNotExcluded(compartmentMemberType, "actor", Location.class.getSimpleName(), searchParameters);
+            addSearchParameterIfNotExcluded(compartmentMemberType, "actor", Practitioner.class.getSimpleName(), searchParameters);
+        } else if (compartmentMemberType.equals(ServiceRequest.class.getSimpleName())) {
+            addSearchParameterIfNotExcluded(compartmentMemberType, "performer", Organization.class.getSimpleName(), searchParameters);
+            addSearchParameterIfNotExcluded(compartmentMemberType, "performer", Practitioner.class.getSimpleName(), searchParameters);
+            addSearchParameterIfNotExcluded(compartmentMemberType, "requester", Organization.class.getSimpleName(), searchParameters);
+            addSearchParameterIfNotExcluded(compartmentMemberType, "requester", Practitioner.class.getSimpleName(), searchParameters);
+            addSearchParameterIfNotExcluded(compartmentMemberType, "subject", Location.class.getSimpleName(), searchParameters); 
+        } else if (compartmentMemberType.equals(Specimen.class.getSimpleName())) {
+            addSearchParameterIfNotExcluded(compartmentMemberType, "collector", Practitioner.class.getSimpleName(), searchParameters);
+            addSearchParameterIfNotExcluded(compartmentMemberType, "subject", Location.class.getSimpleName(), searchParameters);
+        } else if (compartmentMemberType.equals(SupplyDelivery.class.getSimpleName())) {
+            addSearchParameterIfNotExcluded(compartmentMemberType, "receiver", Practitioner.class.getSimpleName(), searchParameters);
+            addSearchParameterIfNotExcluded(compartmentMemberType, "supplier", Organization.class.getSimpleName(), searchParameters);
+            addSearchParameterIfNotExcluded(compartmentMemberType, "supplier", Practitioner.class.getSimpleName(), searchParameters);
+        } else if (compartmentMemberType.equals(SupplyRequest.class.getSimpleName())) {
+            addSearchParameterIfNotExcluded(compartmentMemberType, "requester", Organization.class.getSimpleName(), searchParameters);
+            addSearchParameterIfNotExcluded(compartmentMemberType, "requester", Practitioner.class.getSimpleName(), searchParameters);
+            addSearchParameterIfNotExcluded(compartmentMemberType, "subject", Location.class.getSimpleName(), searchParameters);
+            addSearchParameterIfNotExcluded(compartmentMemberType, "subject", Organization.class.getSimpleName(), searchParameters);
+            addSearchParameterIfNotExcluded(compartmentMemberType, "supplier", Organization.class.getSimpleName(), searchParameters);
+        } else if (compartmentMemberType.equals(VisionPrescription.class.getSimpleName())) {
+            addSearchParameterIfNotExcluded(compartmentMemberType, "prescriber", Practitioner.class.getSimpleName(), searchParameters);
+        } 
```

Comment:
This could generate a massive query. I think this should be configuration driven, so it can be tuned separate from the supported ResourceType in the config.

Net: I think this should be carefully QA'd where a specific set of resources related to the Patient are generated, and we look at the generated SQL and potentially open a downstream issue.